docs/60_initialize_vendors.md

```diff
@@ -0,0 +1,425 @@
+# Initialize Vendor Accounts
+
+The DANDI ecosystem relies on vendor services to operate.  So first you will need to set up accounts with the following vendors:
+
+• **Heroku**: Provisions the API componenents
+• **AWS**: Provides storage buckets, as well as domain management, for resources across the DANDI ecosystem. As well as the services (Kubernetes, etc.) for deploying the JupyterHub.
+• **GitHub**: Serves as the authentication provider for accounts across the DANDI ecosystem
+• **Terraform Cloud**: Manages provisioned resources across cloud vendors in a version-controlled manner. 
+• **Netlify**: Deploys production frontend build, as well as staging previews to assist with frontend development
+• **Sentry**: Provides observability and monitoring for API events
```

Comment:
Not sure why, but the bullet points don't properly render as a list.

![image](https://github.com/user-attachments/assets/9ea81306-f538-428b-8bed-a6f4b93b3c37)

Reply:
Updated!